temporal/scripts/generate_metadata.py

```diff
@@ -0,0 +1,95 @@
+# (C) Datadog, Inc. 2023-present
+# All rights reserved
+# Licensed under a 3-clause BSD style license (see LICENSE)
+
+# Script to update metadata.csv based on temporal's code and current
+# `METRIC_MAP` as defined on metrics.py
+# Must be run in an environment that has the integration installed,
+# and passed the go code from temporal's codebase where metrics are defined via stdin, e.g.:
+# cat ${path_to_temporal}/common/metrics/metric_defs.go | hatch run py3.8-1.19:python ./scripts/generate_metadata.py
```

Comment:
nit: to make this easier can't we just use the URL?

- https://github.com/temporalio/temporal/blob/master/common/metrics/metric_defs.go
- https://raw.githubusercontent.com/temporalio/temporal/master/common/metrics/metric_defs.go

Reply:
Also there is no `cat` on Windows 🐱

Reply:
I didn't want to assume that grabbing the metric names from their unreleased master is necessarily what we always want. The example of usage is simply to give the next user of the script enough info on how to get it working.